### PR TITLE
remove transformer layer ID from the top module

### PR DESCRIPTION
**Pull request by RezaYazdaniAminabadi**

### CODE REVIEW

File: deepspeed/pt/deepspeed_cuda.py [hunk 1]
```diff
@@ -403,11 +403,14 @@ class DeepSpeedTransformerLayer(nn.Module):
 
             initial_biases: Optional: Only used for unit test
     """
-    def __init__(self, layer_id, config, initial_weights=None, initial_biases=None):
+    layer_id = 0
+
```
Comment (chunyang-wen) [inline deepspeed/pt/deepspeed_cuda.py:407]: Docstring should be updated accordingly.

Comment (RezaYazdaniAminabadi) [reply deepspeed/pt/deepspeed_cuda.py:407]: sure, I will do that. thanks

File: deepspeed/pt/deepspeed_cuda.py [hunk 2]
```diff
@@ -393,21 +393,24 @@ def backward(ctx, grad_output):
 class DeepSpeedTransformerLayer(nn.Module):
     """Initialize the DeepSpeed Transformer Layer.
 
+        Static variable:
```
Comment (eltonzheng) [inline deepspeed/pt/deepspeed_cuda.py:396]: The API doc (https://deepspeed.readthedocs.io/en/latest/kernel.html) will auto retrieve the strings here, if "layer_id" is invisible to users,  shouldn't add it here.

Comment (RezaYazdaniAminabadi) [reply deepspeed/pt/deepspeed_cuda.py:396]: good point. I will remove it then

**Pull request closed without merging**